modules/workflow-service-api/src/main/java/org/opencastproject/workflow/api/WorkflowServiceDatabase.java

```diff
@@ -70,6 +70,29 @@ public interface WorkflowServiceDatabase {
    */
   List<WorkflowInstance> getWorkflowInstances(int limit, int offset) throws WorkflowDatabaseException;
 
+  /**
+   * Gets latest workflow instances for all events.
+   * Selects workflows which haven't been finished (DateCompleted IS NULL) or greatest DateCompleted
+   *
+   * @param limit
+   *          max number of workflows to be returned
+   * @param offset
+   *          only return workflows from this point onwards
+   * @return list of all {@link WorkflowInstance}s
+   * @throws WorkflowDatabaseException
+   *           if there is a problem communicating with the underlying data store
+   */
+  List<WorkflowInstance> getLatestWorkflowInstances(int limit, int offset) throws WorkflowDatabaseException;
+
+  /**
+   * Gets the total number of latest workflows for all events.
+   *
+   * @return the number of latest workflows
+   * @throws WorkflowDatabaseException
+   *           if there is a problem communicating with the underlying data store
+   */
+  int getLatestWorkflowCount() throws WorkflowDatabaseException;
```

Comment:
Maybe easier to understand?

```suggestion
   * Returns the number of events workflows have been run on.
   *
   * @return the number of latest workflows
   * @throws WorkflowDatabaseException
   *           if there is a problem communicating with the underlying data store
   */
  int countMediaPackages() throws WorkflowDatabaseException;
```